docs/src/languages/nodejs/nvm.md

```diff
@@ -25,13 +25,7 @@ hooks:
         curl -o- https://raw.githubusercontent.com/creationix/nvm/$NVM_VERSION/install.sh | dash
         export NVM_DIR="$PLATFORM_APP_DIR/.nvm"
         [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-        nvm current
```

Comment:
Why do we need to remove this line?

Reply:
I believe all it does is print the version. In this case `current` is what I noticed. Purely informational.